spec/requests/api/v1/events/events_spec.rb

```diff
@@ -1,5 +1,5 @@
 require 'rails_helper'
-
+require 'pry'
```

Comment:
You shouldn't need pry here. A) you already have `pry-rails` in your Gemfile and B) we shouldn't ship code with pry in it.